docs/en/migrations.rst

```diff
@@ -193,6 +195,11 @@ Queries can be executed with the ``execute()`` and ``query()`` methods. The
                 // query()
                 $stmt = $this->query('SELECT * FROM users'); // returns PDOStatement
                 $rows = $stmt->fetchAll(); // returns the result as an array
+
+                // using prepared queries
+                $count = $this->execute('DELETE FROM users WHERE id = ?', [5]);
+                $stmt = $this->query('SELECT * FROM users WHERE id > 5'); // returns PDOStatement
```

Comment:
Is this meant to be parameterized like the previous statement?

Reply:
Yes 🤦. Fixed in #1994.